src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String DEFAULT_FIGURE_COLOR = Color.WHITE.name();
+    private static final int DEFAULT_CIRCLE_RADUIS = 10;
+    private static final int NUMBER_OF_FIGURE_TYPES = 5;
+    private static final int SIDE_SIZE_LIMIT = 101;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final int paramsRandomizer1;
+    private final int paramsRandomizer2;
+    private final String randomColor;
+
+    public FigureSupplier() {
+        paramsRandomizer1 = random.nextInt(SIDE_SIZE_LIMIT);
+        paramsRandomizer2 = random.nextInt(SIDE_SIZE_LIMIT);
+        randomColor = colorSupplier.getRandomColor();
+    }
+
+    public Figure getRandomFigure() {
+        switch (random.nextInt(NUMBER_OF_FIGURE_TYPES)) {
+            case 0:
+                return getRandomCircle();
```

Comment:
```suggestion
                return new Circle(randomColor, randomSize());
```
Make the same changes for other cases

Where 
```
private double randomSize() {
        return random.nextInt(SIDE_SIZE_LIMIT);
    }
```